src/pages/public/StagingServerOverview/components/ServerOverview.tsx

```diff
@@ -20,11 +20,11 @@ const ServerOverview = (): JSX.Element => {
     getServers();
   }, []);
 
-  const tableHeadClass = "px-5 py-3 border-gray-200 bg-gray-300 text-left text-m font-bold text-gray-600 uppercase tracking-wider";
+  const tableHeadClass = "px-5 py-3 border-gray-200 bg-gray-300 text-left text-m font-bold text-gray-600 uppercase tracking-wider dark:bg-gray-700 dark:text-gray-300";
 
   return (
-    <div>
-      <table className="justify-center m-auto my-20">
+    <div className="overflow-x-auto my-20 mx-10 justify-center dark:bg-gray-500 dark:text-gray-300 rounded-md">
+      <table className="tw-full">
```

Comment:
Styling issue, table not filling container: 
![image](https://user-images.githubusercontent.com/1348830/196730172-abc866eb-37e4-48b6-9afb-761e24686ace.png)

```suggestion
      <table className="w-full">
```